packages/aspell.rb

```diff
@@ -3,21 +3,13 @@
 class Aspell < Package
   description 'GNU Aspell is a Free and Open Source spell checker designed to eventually replace Ispell.'
   homepage 'http://aspell.net/'
-  version '0.60.7-rc1-3'
-  source_url 'ftp://alpha.gnu.org/gnu/aspell/aspell-0.60.7-rc1.tar.gz'
-  source_sha256 '86b5662f24316142f70c5890787bdc5596625ca3604dfe85926ee61f27f2365e'
+  version '0.60.7-1'
+  source_url 'ftp://ftp.gnu.org/gnu/aspell/aspell-0.60.7.tar.gz'
```

Comment:
Can we change this to `https://ftpmirror.gnu.org/aspell/aspell-0.60.7.tar.gz`?  I wish ftp would just go away.

Reply:
It looks like there is an update to the english dictionary also at https://ftpmirror.gnu.org/aspell/dict/en/aspell6-en-2018.04.16-0.tar.bz2.  Can you please update aspell_en.rb also?

Reply:
Actually, don't worry about it.  I'll update in the pre-built binaries PR.